addons/mail/wizard/mail_compose_message.py

```diff
@@ -115,7 +115,8 @@ def default_get(self, fields_list):
     composition_batch = fields.Boolean(
         'Batch composition', compute='_compute_composition_batch')  # more than 1 record (raw source)
     model = fields.Char('Related Document Model', compute='_compute_model', readonly=False, store=True)
-    model_is_thread = fields.Boolean('Thread-Enabled', compute='_compute_model_is_thread')
+    model_id = fields.Many2one('ir.model', 'Related Document Model ID', compute='_compute_model_id')
```

Comment:
ir.model is an admin only field so it would be better to avoid adding and using it, you are going to face acl issues anyway :)